doc/manuals/user/ngsiv2_implementation_notes.md

```diff
@@ -289,6 +290,24 @@ Apart from the `status` values defined for subscription in the NGSIv2 specificat
 
 [Top](#top)
 
+## `forcedUpdate` option
+As extra URI param option to the ones included in the NGSIv2 specification, Orion implements forcedUpdate, 
+than can be used to specify that an update operation have to trigger any matching subscription (and send 
+corresponding notification) no matter if there is an actual attribute update or not. Remember that the 
+default behaviour (i.e. without using the forcedUpdate URI param option) is to updated only if attribute 
+is effectively updated.
+
+The following requests can use the forcedUpdate URI param option:
+* `POST /v2/entities/E?options=forcedUpdate`
```

Comment:
Looking operations in detail, I  have realized that some of them are not correct, as /attrs is missing in the URL.

```
* `POST /v2/entities/E/attrs?options=forcedUpdate`
* `POST /v2/entities/E/attrs?options=append,forcedUpdate`
* `PUT /v2/entities/E/attrs?options=forcedUpdate`
* `PUT /v2/entities/E/attrs/A?options=forcedUpdate` <--- this one is ok
* `PUT /v2/entities/E/attrs/A/value?options=forcedUpdate` <-- this one is ok
* `PATCH /v2/entities/E/attrs?options=forcedUpdate`
```

Please, adjust also the `echo "..."` lines in the .test, when aplicable.

Reply:
fixed in 3150a8a